go/labels/shard.go

```diff
@@ -97,3 +118,28 @@ func maybeOne(set pf.LabelSet, name string) (string, error) {
 		return v[0], nil
 	}
 }
+
+func parsePorts(set pf.LabelSet) (map[uint16]*PortConfig, error) {
+	var out = make(map[uint16]*PortConfig)
+
+	for _, value := range set.ValuesOf(ExposePort) {
+		var portNumber, err = strconv.ParseUint(value, 10, 16)
+		if err != nil {
+			return nil, fmt.Errorf("parsing value '%s' of label '%s': %w", value, ExposePort, err)
+		}
+		if portNumber == 0 || portNumber > 65535 {
```

Comment:
Nope, it should be fine. You still need permissions to bind to ports < 1024, but it's pretty normal for containers to do that, since the default root user has permission to do so.